src/Tools/Github/GitMergeBot/Program.cs

```diff
@@ -83,7 +83,13 @@ private Program(Options options)
             }
 
             var newBranchName = await MakePrBranch(_options.SourceUser, _options.RepoName, remoteIntoBranch, newBranchPrefix);
-            await SubmitPullRequest(newBranchName);
+            var pullRequest = await SubmitPullRequest(newBranchName);
+
+            // pullRequest could be null if we are running in debug mode.
+            if (pullRequest != null)
+            {
+                await CommentOnIssue(_options.SourceUser, _options.RepoName, pullRequest.Number, "@dotnet-bot test vsi please");
```

Comment:
Only pass in the `pullRequest` object and let the method pull out its `Number` property as necessary.  Also, `_options` is an instance field that the method can directly use.